common/src/main/java/net/neoforged/gradle/common/util/run/RunsUtil.java

```diff
@@ -37,21 +42,32 @@ public static String createTaskName(final Run run) {
     public static String createTaskName(final String prefix, final Run run) {
         return createTaskName(prefix, run.getName());
     }
-    
+
+    @Nullable
+    public static Run get(Project project, String runName) {
+        Object runsExtension = project.getExtensions().findByName(RunsConstants.Extensions.RUNS);
+        if (runsExtension instanceof NamedDomainObjectContainer) {
+            return (RunImpl) ((NamedDomainObjectContainer<?>) runsExtension).findByName(runName);
+        }
+        return null;
+    }
+
     public static Run create(final Project project, final String name) {
         final RunImpl run = project.getObjects().newInstance(RunImpl.class, project, name);
         
         final TaskProvider<RunExec> runTask = project.getTasks().register(createTaskName(name), RunExec.class, runExec -> {
             runExec.getRun().set(run);
         });
-        
-        project.afterEvaluate(evaluatedProject -> runTask.configure(task -> {
-            addRunSourcesDependenciesToTask(task, run);
-            
-            run.getTaskDependencies().forEach(task::dependsOn);
-        }));
-        
-        run.getEnvironmentVariables().put("MOD_CLASSES", buildGradleModClasses(run.getModSources()));
+
+        project.afterEvaluate(evaluatedProject -> {
+            run.getEnvironmentVariables().put("MOD_CLASSES", buildGradleModClasses(run.getModSources()));
+
+            runTask.configure(task -> {
+                addRunSourcesDependenciesToTask(task, run);
+
+                run.getTaskDependencies().forEach(task::dependsOn);
+            });
+        });
```

Comment:
This change is not needed as far as I can tell. If you have a reason for it, please document it accordingly.